go/pserver/optimizer.go

```diff
@@ -1,42 +1,71 @@
 package pserver
 
 /*
-#include "optimizer.h"
+// TODO(zhihong): move compile flags to cmake go_library
+#cgo pkg-config: protobuf
+#cgo CFLAGS: -I ../../
+#cgo LDFLAGS: /Users/dzh/.go/src/github.com/PaddlePaddle/Paddle/build/go/pserver/cclient/libpaddle_go_optimizer.a -lstdc++
```

Comment:
`/Users/dzh/.go/src/github.com` need to be a relative path: `#cgo LDFLAGS: -L${SRCDIR}/xxx`